ansible/readme.md

```diff
@@ -41,10 +45,24 @@ too old.
 
 ## Useful commands
 
-Note: deploying assumes there is a `x86_64-unknown-linux-gnu` binary available in
-`../ci-bench-runner/target/release` (generated by `cargo build --release`).
-
 - Setup everything from scratch and deploy: `ansible-playbook playbook.yml`
 - Update Rust to the latest stable version: `ansible-playbook playbook.yml --tags rustup`
 - Deploy a new version of the server: `ansible-playbook playbook.yml --tags deploy`
 - Check the status of the application: `sudo systemctl status ci-bench-runner`
+
+## Testing
+
+To deploy to a test server, copy `inventory.ini` to a new file `test-inventory.ini`, and update 
+the `ansible_host` IP address in `test-inventory.ini`.
+
+You can then deploy using:
+```bash
+ansible-playbook \
+  --inventory test-inventory.ini \
+  --user root \
+  --extra-vars 'hostname=rustls-bench.example.com' \
+  --extra-vars 'github_repo_owner=example' \
+  --extra-vars 'bench_app_repo=https://github.com/example/rustls-bench-app/' \
+  --extra-vars 'bench_app_branch=example-branch' \
+  playbook.yml
+```
```

Comment:
> Is this about testing that the Ansible bits are working properly, or also about testing that the application works after the deploy? In the second case, I think a user would need a bit more guidance by mentioning things like:

I was thinking a bit of both! Mainly I was looking ahead to the days where we have this code running in a "production" deploy on the dedicated server. If we need to make any substantive changes to the application code or to the ansible scripts we'll probably want to test it without impacting the production deploy. I don't think it makes sense to have a whole second server always laying around for this testing, but it would be helpful if we had instructions for setting one up on-demand.

> In the second case, I think a user would need a bit more guidance by mentioning things like:

These are good points. I'll try to add some more text with the points you describe. I think having them written down in the README or somewhere equivalent will be helpful long term.

> It almost feels like we should have a terraform script to create the GitHub forks, the application and its installation 😛... But let's not go down that path.

Haha yeah I think that's probably overkill for now :fearful: (but would be cool!)

Reply:
> These are good points. I'll try to add some more text with the points you describe.

I updated the testing section of the ansible README to cover the motivation, and the required up front setup.